packages/react/src/utils/floating-ui/useClick.ts

```diff
@@ -112,7 +112,8 @@ export function useClick(context: FloatingRootContext, props: UseClickProps = {}
           open &&
           toggle &&
           (dataRef.current.openEvent && stickIfOpen
-            ? dataRef.current.openEvent.type === 'click'
+            ? dataRef.current.openEvent.type === 'click' ||
```

Comment:
@atomiks, I just added this to fix an issue with tests (https://app.circleci.com/pipelines/github/mui/base-ui/9809/workflows/65a23a64-619c-4531-897d-d40332476f6b/jobs/94635).

Reply:
No, it's unrelated. I'm figuring it out.